modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java

```diff
@@ -254,8 +254,36 @@ private void _sendNotification(long companyId) {
 							String body = _generateBody(
 								strAccessTokenExpirationDate);
 
-							_sendNotificationEvent(users, body);
-							_sendEmail(company, subject, body, users);
+
+
+
+							String defaultEmailFromAddress = "scim-notification@" + company.getMx();
+							String defaultEmailFromName = "SCIM-Notification";
+
+							String mail = users.get(0).getEmailAddress();
+
+							SubscriptionSender subscriptionSender = new SubscriptionSender();
+
+							//subscriptionSender.setClassName();
+							//subscriptionSender.setClassPK();
```

Comment:
Here also

Reply:
The comments are in previos commits but in the [final version](https://github.com/liferay-appsec/liferay-portal/blob/621ddc6205b430059cb616660b8af203fdfc6d32/modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java) is not commented.